core/tests/protractor_desktop/additionalEditorFeatures.js

```diff
@@ -75,7 +75,7 @@ describe('Full exploration editor', function() {
       await waitFor.invisibilityOf(
         postTutorialPopover, 'Post-tutorial popover does not disappear.');
       await action.click('State Edit Button', stateEditButton);
-      var stateEditorTag = element(by.tagName('state-content-editor'));
+      var stateEditorTag = element(by.tagName('oppia-state-content-editor'));
```

Comment:
Since we're changing this, can we use a tag with a name of the form `protractor-test-*`? Ditto in `core/tests/protractor_utils/ExplorationEditorMainTab.js`

Reply:
`oppia-state-content-editor` is child component of parent component `state-editor` as element.
In our codebase every where we use this convention for child element.

we generally use `protractor-test-*` for classes with `by.css` not for `element(by.tagname('oppia-*)`

Reply:
Ahh, sorry I missed that this was `by.tagName`. Is it possible to use an HTML class instead? My naïve instinct is that we could add a `protractor-test-*` HTML class [here](https://github.com/oppia/oppia/pull/14838/files#diff-1d7b9599dc8fceec2de846927be3c6691c9de25ded069f01cb4d7cb34c45e588R18-R24)

Reply:
Done